server/src/infra/auth_service.rs

```diff
@@ -314,6 +333,55 @@ where
     get_login_successful_response(&data, &name).await
 }
 
+async fn simple_login<Backend>(
+    data: web::Data<AppState<Backend>>,
+    request: web::Json<login::ClientSimpleLoginRequest>,
+) -> HttpResponse
+where
+    Backend: TcpBackendHandler + BackendHandler + OpaqueHandler + 'static,
+{
+    let password = &request.password;
+    let mut rng = rand::rngs::OsRng;
+    let opaque::client::login::ClientLoginStartResult { state, message } =
+        match opaque::client::login::start_login(&password, &mut rng) {
+            Ok(n) => n,
+            Err(_) => return HttpResponse::InternalServerError().body(String::new()),
```

Comment:
Empty error string?

Reply:
It returns now "Internal Server Error: " appended with the error